### DISCUSSION TITLE
Help & Tips to Creating a TMDb "TV Shows Aired Yesterday" List?

### DISCUSSION THREAD

**Discussion by metaltermite** [jurialmunkey/plugin.video.themoviedb.helper#1286] [Q&A] [answered]
I am trying to create a TMDb list, and I could use any advice. My goal is to generate a list of English language TV shows that aired yesterday.

Currently I have achieved this using the following rules via `TMDb Helper > Discover > Discover TV Shows > ...`
- Air Date After: T-1
- Air Date Before: T-1
- With Original Language: en
- Match ALL values
- Without Genres: [personal preferences here] 
- Without Keywords: [personal preferences here]

However, I am unsure if those specific date rules are narrowing down content to only display tv shows that have aired yesterday. If anyone could provide advice lmk. Would be appreciated.

**Comment by jurialmunkey** [comment, marked as answer]
As far as I understand, yes, air_date should be the most recently aired date for the tvshow (i.e. the last time an episode aired) whereas first_air_date is the original premiere date (i.e. S01E01) for the tvshow.
